### DISCUSSION TITLE
How to add icon in vuetify components

### DISCUSSION THREAD

**Discussion by tomhoma** [cawa-93/iconify-prerendered#14] [General]
Hi, thank you for the great lib but how I use this icon with vuetify prop
<v-chip
      class="ma-2"
      color="orange"
      text-color="white"
      _**append-icon="mdi-star"**_
    >
      Premium
    </v-chip>

**Comment by cawa-93** [comment]
From [docs](https://vuetifyjs.com/en/api/v-chip/#props-append-icon): `append-icon` may accept `FunctionalComponent`.

So… Somethink like below, I assume
```vue
<script setup>
import IconStar from '@iconify-prerendered/vue-mdi'
</script>

<template>
<v-chip
	:append-icon="IconStar"
></v-chip>
</template>
```
